app/controllers/adopters_controller.rb

```diff
@@ -141,7 +141,7 @@ def adopter_params
                                     [
                                       :adopter_id,
                                       :spouse_name,
-                                      :other_household_names,
+                                      :other_fhold_names,
```

Comment:
Why was this edited?

Reply:
I don't remember. I was using other_household_names in the form. I think this was a mistake.